src/dailyLog/index.html

```diff
@@ -30,12 +30,8 @@
             <!-- Heading: Date -->
             <div class="header-container">
                 <h1>Date:</h1>
-                <div class="button-container">
-                    <!-- button to save log -->
-                    <button type="button" id="save-button">Submit</button>
-                    <!-- button for deleting log -->
-                    <button type="button" id="delete-button">Delete Log</button>
-                </div>
+                <!-- button for deleting log -->
+                <button type="button" id="delete-button">Delete Log</button>
```

Comment:
We should probably add a class "js-delete-button" to select by in the JS rather than just by id